### ISSUE TITLE
Anomaly: Evars generated outside of proof engine...

### ISSUE THREAD

**Issue by andres-erbsen** [coq/coq#10939]
#### Description of the problem

~~~coq
Goal False.
  epose (ltac:(
    let x := fresh in 
    intros x;
    induction x; admit) : forall pf : 7=7, True).
(*
Anomaly
"Evars generated outside of proof engine (e.g. V82, clear, ...) are not supposed to be explicitly given up."
Please report at http://coq.inria.fr/bugs/.
*)
~~~

#### Coq Version

8.9.1

**Comment by mwuttke97** [comment]
I also just stubled into this issue. It can be simplified:

```coq
Goal False.
Proof.
  epose proof ltac:(shelve). (* works *)
  epose proof ltac:(admit). (* anomaly *)
```
My use case for this was the following. I wanted to `pose proof` a lemma application, but I knew that I wouldn't be able to prove a premise (because some other definitions have to be refined first, but I was procrastinating). So that why I used 'admit' here :)

Perhaps a better error message should be shown.

(I am using `Coq v8.11.1`)
